### PR TITLE
 SUPERBALL GAME IN C

### LINKED ISSUE #3165: SuperBall Game In C
### Description

This is an advanced program in C Programming with graphics and use of colors!💯
In this game you need to balance the circular ball on a rectangular plate and prevent it from falling down. Once the ball falls down you will loose a point and till you balance the ball on the plate it will score you some points! 👍 

Try to code and enjoy playing! :)   

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by ikavyajain**
<hr>

## Description 📜

SUPERBALL GAME IN C
This is an advanced program in C Programming with graphics and use of colors!💯
In this game you need to balance the circular ball on a rectangular plate and prevent it from falling down. Once the ball falls down you will loose the game at the very moment and till you balance the ball on the plate you will score two(+2) points! 👍

Try to code and enjoy playing! :)

Fixes #3165

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸
Pic1
Pic2
Pic3
Pic4

<hr>

### CODE REVIEW

File: C_CPP/Competitive Programming/SuperBall_Game_In_C.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                                    { setbkcolor(BLUE);
+                                    cleardevice();
+                                    goto phirse;
+                                    }
+                                    }
+                                    
+                                    
+### OUTPUT:
+
+![WhatsApp Image 2021-09-27 at 22 15 43](https://user-images.githubusercontent.com/89743157/134952838-49518594-f2d5-42e0-871b-190418400b24.jpeg)
```
Comment (deeqakkk) [inline C_CPP/Competitive Programming/SuperBall_Game_In_C.md:162]: give relevant names to all your images

File: C_CPP/Competitive Programming/SuperBall_Game_In_C.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+# SUPERBALL GAME
+
+### CODE:
```
Comment (deeqakkk) [inline C_CPP/Competitive Programming/SuperBall_Game_In_C.md:3]: I will strongly suggest using some code formatter for your code, and also add some relevant comments to make it easy to understand.

**Review by deeqakkk — CHANGES_REQUESTED** [review]

**Comment by ikavyajain** [comment]
Can you tell me this code formatter thing? Is it not understandable?

**Comment by ikavyajain** [comment]
Alright!

**Comment by Saviour1001** [comment]
> In the C_CPP section?

yes.

**Pull request closed without merging**